conf/aggregations.go

```diff
@@ -9,16 +9,10 @@ import (
 	"github.com/alyu/configparser"
 )
 
-var DefaultAggregation = Aggregation{
-	Name:              "default",
-	Pattern:           nil,
-	XFilesFactor:      0.5,
-	AggregationMethod: []Method{Avg},
-}
-
 // Aggregations holds the aggregation definitions
 type Aggregations struct {
-	Data []Aggregation
+	Data               []Aggregation
+	DefaultAggregation Aggregation
```

Comment:
but both SetSingleAgg and SetSingleSchema modify global state. wouldn't that global state stay just as "dirty" as it was before? Your goal is noble but I think it can only be fulfilled by passing the Schemas and Aggregations around instead of having them as globals in the mdata package.